tests/endpoints.test.js

```diff
@@ -0,0 +1,38 @@
+const chai = require('chai');
+const request = require('supertest');
+const { app, server } = require('../server/index.js');
+
+const expect = chai.expect;
+
+describe('GET /about/:ticker endpoint tests', () => {
+  it('should return status code 200', done => {
+    request(app)
+      .get('/about/RSTU')
+      .end((err, res) => {
+        expect(res.statusCode).to.equal(200);
+        expect(res.body).to.be.an('object');
+        done();
+      });
+  });
+  it('should return a JSON object', done => {
+    request(app)
+      .get('/about/RSTU')
+      .end((err, res) => {
+        expect(res.body).to.be.an('object');
+        done();
+      });
+  });
+  it('should return information about the stock', done => {
```

Comment:
Were there other fields you were generating? (ie, founded) Or are those coming from a different micro-service?

Reply:
I wasn't sure if it was necessary or redundant to test every single piece of info coming from the api call